CMakeLists.txt

```diff
@@ -322,6 +322,7 @@ endif()
 if (WITH_SYSTEM_PROVIDED_3PARTY)
   set(GFLAGS_USE_TARGET_NAMESPACE ON)
   find_package(gflags REQUIRED)
+  find_package(utf8cpp REQUIRED)
```

Comment:
Is it really packaged on different systems?..

Reply:
Those who set `WITH_SYSTEM_PROVIDED_3PARTY` should make sure they have it.
https://repology.org/project/utfcpp/versions